public/js/helper.js

```diff
@@ -65,65 +42,50 @@ const copyToClipboard = str => {
 
 function cardToMTGAExport(c, language) {
 	let set = c.set.toUpperCase();
-	if(set == "DOM") set = "DAR"; // DOM is called DAR in MTGA
-	if(set == "CON") set = "CONF"; // CON is called CONF in MTGA
+	if (set == "DOM") set = "DAR"; // DOM is called DAR in MTGA
+	if (set == "CON") set = "CONF"; // CON is called CONF in MTGA
 	let name = c.printed_name[language];
-	let idx = name.indexOf('//');
+	const idx = name.indexOf('//');
 	// Ravnica Splits cards needs both names to be imported, others don't
-	if(idx != -1 && (c.set != 'grn' && c.set != 'rna'))
+	if (idx != -1 && (c.set != 'grn' && c.set != 'rna'))
 		name = name.substr(0, idx - 1);
 	return `1 ${name} (${set}) ${c.collector_number}\n`
 }
 
 function exportMTGA(deck, sideboard, language, lands) {
-	let str = "";
-	for(let c of deck)
-		str += cardToMTGAExport(c, language);
-	if(lands) {
-		for(let c in lands)
-			str += `${lands[c]} ${window.constants.BasicLandNames[language][c]}\n`;
+	let str = deck.map(c => cardToMTGAExport(c, language)).join();
+	if (lands) {
+		str += lands.map(c => `${lands[c]} ${window.constants.BasicLandNames[language][c]}\n`).join();
 	}
-	if(sideboard && sideboard.length > 0) {
-		str += '\n';
-		for(let c of sideboard)
-			str += cardToMTGAExport(c, language);
+	if (sideboard && sideboard.length > 0) {
+		str += '\n' + sideboard.map(c => cardToMTGAExport(c, language)).join();
 	}
 	return str;
 }
 
-function exportToMagicProTools(cardsdb, draftLog, userID) {
-	let str = "";
-	str += `Event #: ${draftLog.sessionID}_${draftLog.time}\n`;
+function exportToMagicProTools(cardsdb, draftLog, userId) {
+	let str = `Event #: ${draftLog.sessionID}_${draftLog.time}\n`;
 	str += `Time: ${new Date(draftLog.time).toUTCString()}\n`;
 	str += `Players:\n`; 
-	for(let c in draftLog.users) {
-		if(c == userID)
-			str += `--> ${draftLog.users[c].userName}\n`;
-		else
-			str += `    ${draftLog.users[c].userName}\n`;
-	}
-	
+	str += draftLog.users.map(c => `${(c == userId ? `-->` : `   `)} ${draftLog.users[c].userName}\n`).join();
 	str += '\n';
 	
 	let boosterNumber = 0;
 	let pickNumber = 1;
 	let lastLength = 0;
-	for(let p of draftLog.users[userID].picks) {
-		if(p.booster.length > lastLength) {
+	for (let p of draftLog.users[userId].picks) {
+		if (p.booster.length > lastLength) {
 			boosterNumber += 1;
 			pickNumber = 1;
-			if(draftLog.setRestriction && draftLog.setRestriction.length === 1)
-				str += `------ ${draftLog.setRestriction[0].toUpperCase()} ------\n\n`;
-			else 
-				str += `------ THB ------\n\n`;
+			// todo: why THB here? Should it be MTGSets.slice(-1)[0] aka MTGSets.last()?
```

Comment:
Could be anything actually. When multiple sets are selected cards are mixed in a single booster so this header doesn't make sense anymore, and it seems that at least MTGProTools doesn't really care about this header as long as its present. I just choose THB because it seemed to work fine.

Reply:
I am fine with this decision. I will replace the comment with your answer.